lib/schedule/data.ex

```diff
@@ -560,4 +642,6 @@ defmodule Schedule.Data do
     |> Csv.parse(parse: &Stop.from_csv_row/1)
     |> Map.new(fn stop -> {stop.id, stop} end)
   end
+
+  # Add connections_by_stop_id(routes, route_patterns, stop_times_by_trip_id)
 end
```

Comment:
(late)question(non-blocking/followup): was this intended to be left in the PR?

Reply:
Ah thanks! That was not intended to be left here, will be removed in https://github.com/mbta/skate/pull/1864